README.md

```diff
@@ -52,6 +52,22 @@ npm start
 
 The dev server should now be running on http://localhost:1234
 
+# The structure
+
+If you've already looked in the `/react-dom` directory or `/react` directory, you might have noticed that they 
+are not empty.
+We've taken the liberty of implementing a skeleton for you of empty functions that are to be implemented.
+
+To stay true to the virtual-dom mindset you will find `VCompositeNode.js` and `VDomNode.js` in the `react-dom` 
+directory. `VDomNode.js` is a "virtual" DOM-node, while the `VCompositeNode` represents a "virtual" react-component node.
+Everything that can be represented in the DOM, such as a `number`, `string`, `div`, `a`, `p` etc. should be a 
+`VDomNode`. Everything else, and by that we mean stateless or functional components should be a `VCompositeNode`.
+
+These "virtual"-nodes can have children, which again are "virtual" nodes. This means that we get a tree-structure
+of nodes known as "the virtual DOM". The "virtual DOM" that we are about to implemented is pretty naive. But 
+nevertheless, the structure is there to extend the implementation with a more advanced reconciliation-algorithm that 
```

Comment:
Føles som om det mangler noe. "the structure is there **to make it easier later** to extend" kanskje?